test/lib/mayaUsd/render/mayaToHydra/cpp/CMakeLists.txt

```diff
@@ -35,8 +35,8 @@ target_sources(${TARGET_NAME}
         testSceneCorrectness.cpp
         testPrimInstancing.cpp
         testPicking.cpp
-        testPointInstancePicking.cpp
-        testUsdNativeInstancePicking.cpp
+        testUsdPointInstancePicking.cpp
+        testUsdPicking.cpp
```

Comment:
The USD native instance picking C++ test code was not specific to native instancing, so renamed it for generality.